source/features/stop-redirecting-in-notification-bar.tsx

```diff
@@ -3,9 +3,13 @@ import delegate from 'delegate-it';
 
 import features from '.';
 
+interface SessionStorage extends Storage {
+	notification_shelf: string;
+}
+
 const hasNotificationBar = (): boolean =>
 	location.search.startsWith('?notification_referrer_id=') ||
-	JSON.parse(sessionStorage.notification_shelf ?? '{}').pathname === location.pathname;
+	JSON.parse((sessionStorage as SessionStorage).notification_shelf ?? '{}').pathname === location.pathname;
```

Comment:
Use .getItem instead if this method doesn’t work as well anymore. The only reason why some code is written in a certain way is to make it shorter or more readable. Having to use assertions defeats the purpose of using property access